### PR TITLE
Movie WebApp

### PR DESCRIPTION
**Pull request by theBwoyProgrammer**
# Built a movie web App
- We used HTML/CSS and JavaScript
- Displayed Data from more than one APIs
- Added likes and comment counters
- Tested our functions with JEST

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,59 @@
...
+👤 Hatim
+
+- GitHub: [@HatimDev](https://twitter.com/HatimDev)
+- Twitter: [@HatimDev-HE](https://github.com/HatimDev-HE)
+- LinkedIn: [@hatimdev](https://www.linkedin.com/in/hatimdev/)
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```
Comment (BenMukebo) [inline README.md:80]: - [ ] I would also recommend adding the `Acknowledgments` and the `Contributing` sections with the issue link in : 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).


## Acknowledgments

- Microverse
```



- Don't forget to add the ``` ##  License ``` section at the end of your README file to make it more professional. I would also recommend you add this. 😄 


- please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.

**Review by BenMukebo — CHANGES_REQUESTED** [review]
## Hi @EyasuTeshome, @HatimDev-HE, 

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


### To Highlight 🎉

- Git flow is followed. :heavy_check_mark:
- You have followed  JavaScript best practices. ✔️ 
- PR has a good title and summary. ✔️
- Good presentation video. You looked professional and a great close teammate, I love this team  ✔️ 
- Your app has been well deployed using GitHub Pages. ✔️ 
- Your Design looks awesome, keep rocking :rocket: 

## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Great job recording the video 🎥  presenting the features of the project 👏🏻 But let's make sure that everyone can read the video without having to download the it 👍🏻 I was not able to see it until I download it on my computer

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,59 @@
...
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+To work on this project locally you need to have Node.js installed in your machine.
+
+### Setup
+
+To setup this project first you should have to clone the repository and the go to the projects directory and run npm install command to install the needed dependency. After that run npm start command to run the project from local server to http://localhost:8080/.
+
```
[the remark this answers is not in the document]
Comment (HatimDev-HE) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (HatimDev-HE) [reply README.md:80]: done

**Review by codecaiine — APPROVED** [review]
Hi @EyasuTeshome @HatimDev-HE , 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**